cumulus/parachains/runtimes/people/people-rococo/src/xcm_config.rs

```diff
@@ -169,13 +169,17 @@ impl Contains<RuntimeCall> for SafeCallFilter {
 
 		matches!(
 			call,
-			RuntimeCall::PolkadotXcm(pallet_xcm::Call::force_xcm_version { .. }) |
-				RuntimeCall::System(
-					frame_system::Call::set_heap_pages { .. } |
-						frame_system::Call::set_code { .. } |
-						frame_system::Call::set_code_without_checks { .. } |
-						frame_system::Call::kill_prefix { .. },
-				) | RuntimeCall::ParachainSystem(..) |
+			RuntimeCall::PolkadotXcm(
+				pallet_xcm::Call::force_xcm_version { .. } |
+					pallet_xcm::Call::force_default_xcm_version { .. }
+			) | RuntimeCall::System(
+				frame_system::Call::set_heap_pages { .. } |
+					frame_system::Call::set_code { .. } |
+					frame_system::Call::set_code_without_checks { .. } |
+					frame_system::Call::authorize_upgrade { .. } |
```

Comment:
Maybe the ability to upgrade the runtime like this vs just setcode deserves a mention in the changelog/prdoc, WDYT?

Reply:
This is the normal way to upgrade a parachain though. It's just the move of this call from Parachain System (all of which is included) to System (which specifies certain calls). So it's just preserving functionality.